packages/strapi-admin/admin/src/themes/colors.js

```diff
@@ -35,6 +35,27 @@ const colors = {
     'blue-dark': '#151c2e',
     blue: '#0097f7',
   },
+  filters: {
+    border: '#e3e9f3',
+    background: '#ffffff',
```

Comment:
this should not exist as the white color is already defined in the colors object.